libraries/vmmanager/include/koinos/vmmanager/context.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <koinos/vmmanager/api_handler.hpp>
+
+#include <cstdint>
+
+namespace koinos::vmmanager {
```

Comment:
Rename this namespace `koinos::vmmanager` -> `koinos::vm_manager`.

Reply:
I filed a separate issue #462 for this change, as we'd want to change `statedb` -> `state_db` as well for consistency.